docs/UserGuide.md

```diff
@@ -142,87 +177,136 @@ Format: `cfilter KEYWORD [MORE_KEYWORDS]...`
 
 Edits an existing contact with the specified index in Teaching Assistant.
 
-Format: `cedit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]...`
+**Format**: `cedit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [t/TAG]...`
+
+**Breakdown**:
+* Command word - `cedit`
+* Prefixes - `n/`, `p/`, `e/`, `t/`
+* Parameters - `INDEX`, `NAME`, `NUMBER`, `EMAIL`, `TAG`
+
 
 * `INDEX` refers to the index numer shown in the displayed entry list.
 * `INDEX` must be a positive integer 1,2,3, ...
 * At least one of the optional fields must be provided.
 * Existing values will be updated to the input values.
 * When editing tags, the existing tags of the contact will be removed i.e. adding of tags is not cumulative.
-* You can remove all the contact’s tags by typing t/ without specifying any tags after it.
+* You can remove all the contact’s tags by typing `t/` without specifying any tags after it.
 
 Example(s):
 
 * `cedit 1 p/91234567 e/alexyeoh@example.com` Edits the phone number and email address of the contact corresponding to 
   index 1 and to `p/91234567` and `alexyeoh@example.com` respectively.
 * `cedit 1 n/Bernice Yu Xiao Ling t/` Edits the name of the contact corresponding to index 1 to be `Bernice Yu Xiao Ling` 
   and clears all existing tags.
+  
+Scenario:
+Your student had recently changed his/her phone number and you want to update his/her contact details without the 
+hassle of deleting his/her old contact and then subsequently adding a new updated contact.
 
-### Listing contacts
+### Listing contacts: `clist`
 
 Lists all the contacts in Teaching Assistant.
 
-Format: `clist`
+**Format**: `clist`
+
+**Breakdown**:
+* Command word - `clist`
 
-### Deleting a contact
+Scenario: You want to look at all the contacts you have saved.
+
+### Deleting a contact: `cdelete`
 
 Deletes an existing contact with the specified index in Teaching Assistant.
 
-Format: `cdelete INDEX`
+**Format**: `cdelete INDEX`
+
+**Breakdown**:
+* Command word - `cdelete`
+* Parameters - `INDEX`
 
-* `INDEX` refers to the index number shown in the displayed person list.
+
+* `INDEX` refers to the index number shown in the displayed contact list.
 * `INDEX` must be a positive integer 1, 2, 3, ...
 
 Example(s):
 
 * `cdelete 1`
 
+Scenario: Your students have long graduated and you will not be contacting them anymore. You want to remove their
+contacts to make space for the contacts of your new students.
+
 ---
 
-### Adding an entry
+### Adding an entry: `eadd`
 
 Adds a new entry into Teaching Assistant.
 
-Format: `eadd n/NAME sd/START_DATE ed/END_DATE [t/TAG]...`
+**Format**: `eadd n/NAME sd/START_DATE ed/END_DATE [t/TAG]...`
+
+**Breakdown**:
+* Command word - `eadd`
+* Prefixes - `n/`, `sd/`, `ed/`, `t/`
+* Parameters - `NAME`, `START_DATE`, `END_DATE`, `TAG`
+
 
 * `START DATE` and `END DATE` are in the format `yyyy-mm-dd HH:MM`.
 * `START DATE` should be before `END DATE`.
 * Entries cannot overlap. i.e. Entries with overlapping timings will not be added.
 
```

Comment:
May need to add an extra thing saying overdue entries can still be added but the entry card will be shown as red, I'll define what an entry card means at my part

Reply:
Sry for the scattered reviews ^^" I'm noticing stuff as i'm doing my part

Reply:
hahah no problems :thumbsup: